exchange/exchange.go

```diff
@@ -99,6 +99,16 @@ func (big *bidIDGenerator) New() (string, error) {
 	return rawUuid.String(), err
 }
 
+type deduplicateBidBooleanGenerator interface {
+	New() bool
```

Comment:
Consider renaming `New` to `Generate` to pair with the "Generator" part of the struct name.

Reply:
Renamed